spec/classes/agent_spec.rb

```diff
@@ -88,6 +88,18 @@
         end
       end
 
+      context 'when binary_location is define', if: facts[:kernel] == 'Linux' do
+        let :params do
+          { binary_location: '/usr/sbin/zabbix_agentd' }
+        end
```

Comment:
Technically this is redundant. If the kernel is Linux, this should already be present so you could drop this.